src/components/Dashboard/TodoList.vue

```diff
@@ -796,6 +834,72 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     }
   }
 
+  /** check if the item is actually affiliation invite todo; (not a regular filing item). */
+  isAffiliationInvitation (item): boolean {
+    // check that affiliation invitation details are set
+    return !!item.affiliationInvitationDetails
+  }
+
+  authorizeAffiliationInvitation (isAuthorized, affiliationInvitationTodo): void {
+    AuthServices.authorizeAffiliationInvitation(
+      this.getAuthApiUrl,
+      this.getIdentifier,
+      affiliationInvitationTodo.affiliationId,
+      isAuthorized
+    )
+      .then()
+      .catch(err => {
+        // eslint-disable-line no-console
+        console.log('failed the call for authorization of affiliation invitation', err)
+      })
+  }
+
+  /** Loads Org Affiliations invitation todo **/
+  async loadAffiliationInvitationsTodo () {
+    // feature-flag-it
+    if (!GetFeatureFlag('enable-affiliation-invitation-request-access')) {
+      return
+    }
+
+    function buildTodoItemIfFromAffiliationInvitation (affiliationInvitation, order: number) {
+      const newTodo: TodoItemIF = {
+        draftTitle: null,
+        enabled: true,
+        filingId: -1, // not a filing
+        name: null,
+        order: order,
+        subtitle: `From: ${affiliationInvitation.fromOrg.name}`,
+        status: null,
+        title: 'Request for authorization to manage this business',
+        affiliationInvitationDetails: {
+          id: affiliationInvitation.id,
+          fromOrgName: affiliationInvitation.fromOrg.name,
+          additionalMessage: affiliationInvitation.additionalMessage
+        }
+      }
+
+      return newTodo
+    }
+
+    // load all the invitations here and push them into todo items
+    const response =
+      await AuthServices.fetchAffiliationInvitations(this.getAuthApiUrl, this.getIdentifier)
+        .catch((err) => {
+          console.log('Error fetching affiliation invitations for todo', err) // eslint-disable-line no-console
```

Comment:
Should this be a console.error or console.warn ?

Reply:
Or a dialog to let the user know?

Reply:
UX has no specific scenario for this, when I previously asked they said we could potentially display a generic modal for that page.

I could provide alertbox / modal saying something like following:

"Failed to fetch affiliation invite TODOs." 

Would that work ?
Or if someone can suggest example how failed API calls are handled right now ? 
I would prefer not to fail loading entire page for it though, as it might retrieve other TODO items.

Reply:
The code has numerous examples of a dialog for API errors.

As this isn't a user error, I'd be ok with an alert() as a last resort, but I'd prefer to see a dialog. You probably need to create a new one as this is a specific error.

Reply:
Agreed, dialog looks much better and more user friendly.

Added preview of how the dialog looks like.
<img width="562" alt="image" src="https://github.com/bcgov/business-filings-ui/assets/7550170/209c8434-94f6-4730-944a-9625a96634ca">

Also, I named it "Generic error dialog" so any error like this could reuse it by using template slots. Added title and message, and example inside the dialog component if no title and message is displayed.  
Hope that this is acceptable.

Reply:
I think this looks great! Did you happen to check with UX folks if they like it too?

Reply:
OK, apparently this is old design. I updated it to match new design. Also changed it a bit so that it should always conform to their style (e.g. can change the icon, but it is not a slot anymore, just icon mdi font, etc ...)

New design looks like this:
<img width="553" alt="image" src="https://github.com/bcgov/business-filings-ui/assets/7550170/14d632e7-f1b2-42cd-8504-09e8f004a687">

Reply:
Looks good to me.

Kial, OK to resolve this conversation?